utils/rpms/daos.spec

```diff
@@ -582,6 +582,10 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Fri Feb 02 2024 Ashley M. Pittman <ashley.m.pittman@intel.com> 2.3.103-15
+- Update pydaos install process
+- Add a depency from daos-client-tests to daos-devel and gdb
```

Comment:
```suggestion
- Add a dependency from daos-client-tests to daos-devel and gdb
```

Reply:
Hm. I thought I'd pushed a fix for this.  Re-pushed.